tests/unit/libbpf_test.cpp

```diff
@@ -3233,14 +3233,68 @@ TEST_CASE("bind_tail_call_max_exceed", "[libbpf]")
     usersim_trace_logging_set_enabled(false, 0, 0);
 }
 
-TEST_CASE("libbpf map batch", "[libbpf]")
+void
+_test_batch_iteration_maps(fd_t& map_fd, uint32_t batch_size, bpf_map_batch_opts* opts, bool concurrent_delete)
```

Comment:
The concurrent_delete flag is never passed as true. What purpose does it serve?

Reply:
Don't add new code with known bugs.   If there's future work to use something, don't add it until then.
Otherwise, there's no way for us to review that having such a parameter is the right way to do something.